packages/bot-web-ui/package.json

```
@@ -22,7 +22,7 @@
     "build": "f () { npm run build:skeleton && webpack --optimize-minimize --progress --env.base $1;}; f",
     "build:skeleton": "lerna exec --scope @deriv/bot-skeleton -- npm run build",
     "build:travis": "echo \"No build:travis specified\"",
-    "test": "npm run test:eslint && npm run test:mocha",
+    "test": "npm run test:eslint && npm run test:mocha && npm run test:stylelint",
```

Better to put mocha at the end, in case any of the linting steps fail, the tests will not be executed. This will save us time. Will commit it in and push the card. 👍 

```suggestion
    "test": "npm run test:eslint && npm run test:stylelint && npm run test:mocha",
```